src/doc/unstable-book/src/language-features/no-sanitize.md

```diff
@@ -1,29 +0,0 @@
-# `no_sanitize`
-
-The tracking issue for this feature is: [#39699]
-
-[#39699]: https://github.com/rust-lang/rust/issues/39699
-
-------------------------
-
-The `no_sanitize` attribute can be used to selectively disable sanitizer
```

Comment:
This info will need to go into the Reference (probably here? https://github.com/rust-lang/reference/blob/master/src/attributes/codegen.md)

Reply:
Created rust-lang/reference#1701.